### ISSUE TITLE
[Screen Reader - MCIO DCX-O1NLD]: The Sorting information (Sorted ascending/sorted descending) of the table present under '01NLD Land' is not announced NVDA.

### ISSUE THREAD

**Issue by msftedad** [angular-ui/ui-grid#7251]
Description:
Screen reader users will get impacted as the user will not be able to know whether the sorting occurred or not if the screen reader doesn't announce the sorting information. Screen reader users get confused and affected.

Test Environment:
OS Build: Version 21H2(22000.856)
Browser: Microsoft Edge Version 105.0.1343.33 (Official build) (64-bit)
Screen reader: Non-Visual Desktop Access (NVDA) (2022.2)

Repro Steps:
1. Open [Angular Grid | Excel Style Filtering | Infragistics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel-style-filtering)
2. Angular excel style filtering page will open.
3. Reach to the main landmark content where ' Examples' tab is selected.
4. With 6-tab press, reach to the column header of the available table.
5. Verify the issue that the sorting(ascending/descending) information is announced.

Actual Result:
The Sorting information (Sorted ascending/sorted descending) of the table present under '01NLD Land' is not announced NVDA.

Expected Result:
The Sorting information (Sorted ascending/sorted descending) of the table present under '01NLD Land' should be announced NVDA.

Additional Information:
The narrator and JAWS do not announce Sorting information also.
Same Issue Reproes with all the tables present throughout the application.

Attachment:
<img width="958" alt="MAS1 3 1_sorting info not announced" src="https://user-images.githubusercontent.com/93735775/189912719-a7edcce6-a2d7-4f61-9c0c-926f1788b60a.png">

**Comment by msftedad** [comment]
This issue is already logged.
